EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFUnpackerInputs.cc

```diff
@@ -55,7 +55,7 @@ namespace l1t
       //Get header ID and payload from the given Block
       unsigned int blockId = block.header().getID();
       LogDebug("L1T") << "Block ID: " << blockId << " size: " << block.header().getSize();
-      auto payload = block.payload();
+      const auto& payload = block.payload();
```

Comment:
This results in a compiler warning. It seems the variable is unused.
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-23389/28557/build-logs/EventFilter/L1TRawToDigi/log.html

Reply:
Yes in deed, I just 'grep'ed it and is not in use any more, seems like a leftover by the winter upgrades to the unpacker.
I'll modify it and we'will upgrade the PR.